css/font-awesome/css/graphik.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
```

Comment:
Are we using Graphik in the end?

Reply:
No, we will use Inter. This was made with Graphik , it was before we decided to change the font. I'll update